torch/onnx/symbolic_helper.py

```diff
@@ -314,16 +314,27 @@ def _slice_helper(g, input, axes, starts, ends, steps=None, dynamic_slice=False)
         from torch.onnx.symbolic_opset10 import _slice as _slice10
         return _slice10(g, input, axes, starts, ends, steps, dynamic_slice)
 
+_fp_tensor_types = set((torch.float16, torch.float32, torch.float64, torch.bfloat16))
```

Comment:
Let's add more structure:
```suggestion
_TypeGroup = collections.namedtuple("TypeGroup", ("tensor_types", "scalar_types"))
_FPTypeGroup = _TypeGroup((torch.float16, torch.float32, torch.float64, torch.bfloat16), ("Float", "Double", "Half", "BFloat16"))
_BoolTypeGroup = _TypeGroup((torch.bool,), ("Bool",))
```

Reply:
Sure.